### DISCUSSION TITLE
Consejos

### DISCUSSION THREAD

**Discussion by Pablobeg** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#468] [Q&A] [answered]
¿Como puedo hacer mi pagina web responsive? He tenido problemas con eso en las practicas 2 y 3, algun recurso o libro que recomienden para esto por fa 👀🙏

**Comment by hackit92** [comment]
Básicamente con media queries o con flexbox, puedes tener un buen punto de partida aqui.
https://www.w3schools.com/css/css_rwd_mediaqueries.asp

**Reply by Pablobeg** [reply]
Gracias por la info bro

**Comment by FedericoCadena9** [comment, marked as answer]
Debes de utilizar una propiedad en CSS llamada Media queries, estos harán que cualquier código ingresado dentro de las llaves lo implementen al tamaño establecido.

La sintaxis la puedes encontrar como la siguiente:
`@media (min-width: 640px)
{ /*Tu código aquí*/ }`
La consulta anterior se activará solo para pantallas que tiene 640 píxeles minimo de ancho.

Un buen punto de partida para definir tus breakpoints (sin tener que batallar con saber cuál aplicar a cada uno) es guiarte de algún lado, llámese framework, estándar, etc. Personalmente te recomiendo los que maneja Tailwind CSS, están adaptados para iniciar desde el dispositivo móvil  y escalar hasta un ordenador, con esto el Responsive (en cuanto a media queries) estará más que resuleto.:)

<html><body>

Minimum width | CSS
-- | --
640px | `@media (min-width: 640px) { ... }`
 768px | `@media (min-width: 768px) { ... }`
 1024px | `@media (min-width: 1024px) { ... }`
1280px | `@media (min-width: 1280px) { ... }`
1536px | `@media (min-width: 1536px) { ... }`

</body>
</html>

**Reply by Pablobeg** [reply]
Gracias ya tengo una idea c:

**Comment by doguedogue** [comment]
Puedes usar CSS Grid, **no** se requieren media queries.

https://css-tricks.com/auto-sizing-columns-css-grid-auto-fill-vs-auto-fit/

**Reply by Pablobeg** [reply]
Gracias bro
